src/Files.App/CommandLine/CommandLineParser.cs

```diff
@@ -157,12 +157,14 @@ public static string[] SplitArguments(string commandLine, bool trimQuotes = fals
 					int endIndex = argument.IndexOf(':');
 					key = argument.Substring(1, endIndex - 1);   // trim the '/' and the ':'.
 					int valueStart = endIndex + 1;
-					val.Add(valueStart < argument.Length ? argument.Substring(
-						valueStart, argument.Length - valueStart) : null);
+                    if (valueStart < argument.Length)
+                    {
+                        val.Add(argument[valueStart..]);
+                    }
 				}
 				else
 				{
-					key = args[index].Substring(1);
+					key = args[index][1..];
 				}
```

Comment:
The formatting has been broken in this section. Could you fix it?

Reply:
done